README.adoc

```diff
@@ -1,102 +1,405 @@
+    .. Type the URL `chrome://extensions` into the address bar and press kbd:[Enter]
+    .. Type `asciidoctor` into the extension search bar then when the {project-name} block is visible, click its btn:[Details] button.
+    .. Check `Allow access to file URLs`
+ 
+You have installed the extension, and configured what the browser allows it to do.
+Next, review and configure the <<options, options>>.
 
-=== Firefox
+=== Detailed install for Firefox
 
- 1. Install the extension from the {uri-mozilla-addon}[Firefox Add-ons website]
- 2. Open local or remote .ad, .adoc, .asc, .asciidoc files in Firefox
- 3. Enjoy!
+Install the extension from the {uri-mozilla-addon}[Firefox Add-ons website]
 
-[WARNING]
-====
-On Firefox, pages viewed directly from GitHub may not render properly.
-This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the Content Security Policies (CSP).
-The restrictive CSP sent by GitHub are applied to the content inserted by the WebExtension.
+You have now installed the extension.
+Next, review and configure the <<options>>.
 
-In addition, the extension does not automatically trigger in Firefox when navigating to a `file://` from a link.
-Instead, you have to enter the `file://` URL directly into the address bar of the browser.
-If you do navigate to a `file://` URL from a link, simply highlight the address bar and press "Enter" to force the extension to load. There's an open issue in the https://bugzilla.mozilla.org/show_bug.cgi?id=1266960[Firefox bug tracker] to fix this limitation.
-====
+NOTE: There are known issues with Firefox. See <<known-issues>>.
 
-== Installation
+[#example]
+== Example
+The two views below show two images of the {project-name} viewing a simple AsciiDoc, active and inactive.
 
-Loading your local copy of the extension on Chrome (or Chromium), Firefox or Opera is super easy!
+The active extension viewing the HTML from the AsciiDoc is on the left, and after being inactivated, the AsciiDoc being viewed on the right.
 
-=== Chrome, Chromium, Brave or Opera
+You may reproduce these examples yourself by following the instructions further below in <<try-it-yourself>>.
 
- 1. Check `Developer mode` in `chrome://extensions`
- 2. Click `Load unpacked extension...` and select the extension `app` directory
- 3. That's all
+[cols="a,a"]
+|===
+|.Active
+image::{uri-ext-active-image}[width="400px",height="400px"]
 
-To see your changes, click the `Reload (Ctrl+R)` link in `chrome://extensions`.
-If you want to create a `Pack extension` just make a `zip` file of the extension `app` directory.
 
-=== Firefox
+|.Inactive
+image::{uri-ext-inactive-image}[width="400px",height="400px"]
+|===
 
- 1. Click on the `Load Temporary Add-on` in `about:debugging`
- 2. Select the manifest file in the `app` directory
- 3. That's all
+[#new-to-asciidoc]
+== New to AsciiDoc
+AsciiDoc is a plain text markup language.
+
+From the AsciiDoc {uri-asciidoc}[Home page]
+[QUOTE]
```

Comment:
QUOTE is not a valid block style, no is a style even warranted here. Also, always put an empty line above a new block.